app/views/layouts/application.html.haml

```diff
@@ -1,11 +1,29 @@
 !!! 5
 %html
-%head
-  %title Brewery
-  = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
-  = javascript_include_tag 'application', 'data-turbolinks-track' => true
-  = csrf_meta_tags
+  %head
+    %title Brewery
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
 
-%body
+  %body
+    = link_to "Brewery", root_path
+    = link_to "New Recipe", new_recipe_path
 
-  = yield
+    .user-info
+      - if current_user
```

Comment:
Consider using `if user_signed_in?` on this line. It is pretty much the same as `if current_user`, though `if user_signed_in?` makes it a little easier to understand the intention of this section of code for a developer who is unfamiliar with devise's methods.